docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -389,6 +389,10 @@ Let's see how we can configure this.
 
    {{< figure src="/media/tutorials/grafana-alert-on-dashboard.png" alt="A panel in a Grafana dashboard with alerting and annotations configured" caption="Displaying Grafana Alerts on a dashboard" >}}
 
+{{ <admonition type="tip">}}
```

Comment:
This would need to be in an interactive ignore at the moment. We should get admonition support added soon. Unless this is currently a non Killercoda tutorial

Reply:
true! i forgot to add the interactive ignore to this bit.thanks Jay!